### PR TITLE
Fix: Repeat-while-true did not work properly with JS

### PR DESCRIPTION
**Pull request by dmitry-zaitsev**

### CODE REVIEW

File: maestro-client/src/main/java/maestro/Maestro.kt [hunk 1]
```diff
@@ -397,6 +397,8 @@ class Maestro(private val driver: Driver,
     }
 
     fun inputText(text: String) {
+        LOGGER.info("Inputting text: $text")
```
Comment (axelniklasson) [inline maestro-client/src/main/java/maestro/Maestro.kt:400]: Is this change intended?

**Review by axelniklasson — APPROVED** [review]

File: maestro-client/src/main/java/maestro/Maestro.kt (hunk 1 above)
Comment (dmitry-zaitsev) [reply maestro-client/src/main/java/maestro/Maestro.kt:400]: Yes, for some reason we did not have a log message for that command

**Review by artem888 — APPROVED** [review]
